framework/Files.py

```diff
@@ -166,6 +166,14 @@ def getLinkedCode(self):
     """
     return self.__linkedModel
 
+  def getModelType(self):
```

Comment:
Hm, I am curious if we could just use getType instead of adding a model type here?

Reply:
This is possible as well. We would have to specify to users to utilize optional type attributes in an unique format to trigger pyomo printing in rom.py.

For example, specify 'xxxpyomo' format where 'xxx' is an arbitrary name. Of course we would have to check if this does not create other problems with other tools which I am not sure if 'test raven' will capture.

I figured by creating a get method, we could avoid this. What do you think?

Reply:
Yes, I was thinking we could just do ```type="pyomo"```  See also: https://github.com/idaholab/raven/pull/1482/files#r761537840

Reply:
Fixed with commit 313f69a47